lib/cb/requests/resumes/post.rb

```diff
@@ -28,7 +28,8 @@ def body
             privacySetting: args[:privacy_setting],
             resumeFileData: args[:resume_file_data],
             resumeFileName: args[:resume_file_name],
-            hostSite: args[:host_site]
+            hostSite: args[:host_site],
+            entryPath: args[:entry_path]
```

Comment:
Not a blank string, javascript `null`. The API was changed in such a way that if no input param was provided, or one was provided that doesn't line up to one of the `MXResume.eEntryPath` enums, the resulting new resume is marked with the default `APIImport` entrypath.